From: SeongJae Park <sj@kernel.org>
Date: Tue, 2 Dec 2025 22:32:57 -0800
Subject: [PATCH] mm/damon/sysfs: support write-type access sample filter

Allow users utilizing the write/read-only access sample results
filtering, by adding a new supported input for specifying the filter is
for that purpose.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/sysfs.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/mm/damon/sysfs.c b/mm/damon/sysfs.c
index 812a241e602f..67ea5625ddde 100644
--- a/mm/damon/sysfs.c
+++ b/mm/damon/sysfs.c
@@ -782,6 +782,10 @@ damon_sysfs_sample_filter_type_names[] = {
 		.type = DAMON_FILTER_TYPE_THREADS,
 		.name = "threads",
 	},
+	{
+		.type = DAMON_FILTER_TYPE_WRITE,
+		.name = "write",
+	},
 };
 
 static ssize_t type_show(struct kobject *kobj,
-- 
2.47.3

